www/docs/en/dev/config_ref/images.md

```diff
@@ -119,6 +120,45 @@ different screen resolutions.
 - [App icon and image size guidelines](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/IconMatrix.html)
 
 ##Windows
+
+For Windows the recommended approach to define app icons is to use `target` attribute.
+
+```xml
+    <platform name="windows">
+        <icon src="res/windows/storelogo.png" target="StoreLogo" />
+        <icon src="res/windows/smalllogo.png" target="Square30x30Logo" />
+        <icon src="res/Windows/Square44x44Logo.png" target="Square44x44Logo" />
+        <icon src="res/Windows/Square70x70Logo.png" target="Square70x70Logo" />
+        <icon src="res/Windows/Square71x71Logo.png" target="Square71x71Logo" />
+        <icon src="res/Windows/Square150x150Logo.png" target="Square150x150Logo" />
+        <icon src="res/Windows/Square310x310Logo.png" target="Square310x310Logo" />
+        <icon src="res/Windows/Wide310x150Logo.png" target="Wide310x150Logo" />
+    </platform>
+```
+
+where `source` is the path to the icon which needs to be added.
+
+Please note that Windows platform handles MRT icons automatically, so if you specify `src="res/windows/storelogo.png"` the following files will be copied into app's `images` folder: `res/windows/storelogo.scale-100.png`, `res/windows/storelogo.scale-200.png`, etc.
+
+The `target` attribute specifies the base name for resultant icons. For every icon file destination filename is calculated as `target + '.' + MRT_qualifiers + extension(src)`. For the icons to display properly in resultant app every `target` value should be the one of icon filenames, defined in application's `.appxmanifest` file.
+
+Summarizing the above, using `target` attribute it is possible to:
+
+  * define a group of icons for different device scale factors using single `<icon ...>` element, for example:
+```xml
+    <icon src="res/Windows/AppListIcon.png" target="Square44x44Logo.png" />
```

Comment:
It should not, according to the paragraph before:
> The `target` attribute specifies the _base name_ for resultant icons.

This is another copy/paste typo